src/scheduler/simulate.c

```diff
@@ -1099,6 +1099,7 @@ dup_timed_event(timed_event *ote, server_info *nsinfo)
 		return NULL;
 
 	nte = create_event(ote->event_type, ote->event_time, event_ptr, ote->event_func, ote->event_func_arg);
+	set_timed_event_disabled(nte, ote->disabled);
```

Comment:
why not just do "nte->disabled = ote->disabled" ?

Reply:
If there is a setter, why not use it?  It means if I ever need to do something every time the disabled bit is set, I can do it there and trust it will always happen when an event is disabled.

Reply:
But the event should just be duped, right? So if we act on it, it might no longer be a dupe.

Reply:
A good example is the call to create_event().  It handles setting the run_event and end_event pointers to a resource_resv structure.  This is something that needs to be done every time we create an event.  The setter should do everything to disable an event.  If an event is disabled in the old universe, it should be disabled in the new universe.  I don't understand why we shouldn't call it.

Reply:
> if there is a setter, why not use it?

Since the setter doesn't do anything other than assignment, it seemed unnecessary. If you fore-see us adding more code in the future, then cool. The -O2 flag should inline this function anyways, so there shouldn't be any overhead as such.